docs/data_managers/install/configuration.md

```diff
@@ -161,6 +162,13 @@ provided to validate datasets.
         project will need to make this decision during the initial configuration of a
         data system.
 
+**The `maximum_embargo_months` element**
+
+: Data providers can set an embargo date on their datasets. Once published, the metadata
+  for the dataset can be seen on Zenodo but the data itself cannot be downloaded until
+  after the embargo has expired. This configuration element sets the maximum number of
+  months allowed for this embargo period: it defaults to 24 months.
```

Comment:
Maybe make this slightly more explicit that this is a choice made by the data manager as to the maximum length of embargo that data providers can request, at the moment it reads a bit like it's a choice that data managers would blanket make for all datasets

Reply:
blanket in the sense that it reads like data managers are choosing the embargo length for each dataset